apps/filecoin-site/src/app/_components/Navigation/constants.ts

```diff
@@ -12,4 +12,5 @@ export const NAV_LINKS = [
 
 export const TOUCH_TARGET_NAV_LINK = {
   touchAreaPadding: 'px-3 py-2',
```

Comment:
This would make the touch target 48px, the minimum I believe. Currently it's only 40px.

```suggestion
  touchAreaPadding: 'p-3',
```